app/styles/layouts/_l-default.scss

```diff
@@ -190,32 +190,32 @@ html, body {
   @include xy-gutters($callout-padding*2, margin, right left, true);
 
   .indicator-cell--stat {
-    @include xy-cell-static(7, true, $callout-padding*2);
+    @include xy-cell(7, true, $callout-padding*2);
     @include breakpoint(large) {
-      @include xy-cell-static(4, true, $callout-padding*2);
+      @include xy-cell(4, true, $callout-padding*2);
     }
     @include breakpoint(xlarge) {
-      @include xy-cell-static(6, true, $callout-padding*2);
+      @include xy-cell(6, true, $callout-padding*2);
     }
   }
 
   .indicator-cell--borocity {
-    @include xy-cell-static(5, true, $callout-padding*2);
+    @include xy-cell(5, true, $callout-padding*2);
```

Comment:
This was cherry-picked from DH's commit from Apr 2021. I assumed perhaps foundation changed and the -static had to be removed. @dhochbaum-dcp  Do you recall the reason this was done?

Reply:
From the [docs](https://get.foundation/sites/docs/xy-grid.html#xy-cell-static):

>  `xy-cell-static()` is deprecated and will be removed.

Same as :
> xy-cell: $gutter-output is deprecated and will be removed.

but that is still in. I can try to fix/update that, if needed.

Reply:
Ok did a little digging. I don't think we need to change anything here but file this under "If something suddenly looks wonky, this is the first thing to check".

We use `foundation-sites` and the accompanying `ember-cli-foundation-6-sass` library for utility classes in this repo. Currently, our package.json has `"foundation-sites": "^6.3.1"` which means, anytime anyone runs `yarn`, yarn will install the highest available version of the library that is **above** 6.3.1 but **below** 7.0.0 (that's what the carrot means) and update the yarn.lock to reflect the exact version it installed. If you look at the `yarn.lock` at this commit, it says it installed 6.6.3. This is why using that `^x.x.x` syntax in package.json is dangerous unless you're sure that the package you're installing follows [Semantic Versioning](https://semver.org/) (which most popular libraries and frameworks do these days). I looked up these mixins in Foundation's [documentation](https://get.foundation/sites/docs/flexbox-utilities.html#xy-cell-) and it looks like `xy-cell-static` [was marked as deprecated in 6.6.0](https://github.com/foundation/foundation-sites/pull/11081). In this context "deprecated" means "We are warning you that this is going to go away" but it is still there. That explains why this change was probably done - at some point, someone ran `yarn` which moved the version from 6.3 to 6.6. That change doesn't affect how the application actually performs but it would have causes a deprecation warning to pop up in their console. Foundation's [migration notes](https://github.com/foundation/foundation-sites/pull/11405) say to use `xy-cell()` instead, so these changes are correct and should stay. _However_, they say that you may need to pass the argument `$output: (size gutters)` to `xy-cell()` to get it to behave the same as `xy-cell-static()`. Given that we don't really have context into why that method was used originally and what exactly the desired styling is supposed to be, I think we can merge this code in as-is and just keep in mind that if any visual bugs come up in QA, we might need to add that `$output` argument.

It's worth noting that, according to those migration notes, Foundation does sometimes do actual breaking changes in minor (i.e. 6.3-6.6) releases and therefore it appears they do not follow semantic versioning. It just so happens that the mixin we're looking at here isn't affected and was just given a deprecation warning (which was very nice of them :) )